Cargo.toml

```diff
@@ -1,7 +1,9 @@
 [package]
-name = "calcver-cli"
+name = "calcver"
```

Comment:
same name as dependency
see: easy-semver/calcver-rs/issues/15

Reply:
updated deplist to reflect the rename in easy-semver/calcver-rs/pull/16